src/txmempool.h

```diff
@@ -591,7 +586,7 @@ class CTxMemPool
 
     /** The minimum fee to get into the mempool, which may itself not be enough
       *  for larger-sized transactions.
-      *  The minReasonableRelayFee constructor arg is used to bound the time it
+      *  The incrementalRelayFee policy variable is used to bound the time it
```

Comment:
Should still mention what minReasonableRelayFee constructor arg is for --  I guess fee estimation wants the min relay fee?